src/main/java/ymfc/list/RecipeList.java

```diff
@@ -36,6 +36,10 @@ public boolean removeRecipeByName(String name) {
         return false;
     }
 
+    public void removeRecipeByID(int id) {
+        recipes.remove(id);
+    }
+
```

Comment:
Oh, for this it is because I found a use for such a method, so I re-implemented it back again, since I'm using it to remove recipes by ID for my recommended list when printing them.

Reply:
On second thought, this can once again be removed with my new implementation